bin/hbslint.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+let exitCode = 0;
+
+// import packages
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs');
+const chalk = require('chalk');
+const walkSync = require('walk-sync');
+const Linter = require('ember-template-lint');
+const getConfigPath = require('../helpers/get-config-path');
+
+const { argv } = yargs;
+const { log } = console;
+
+// configuration
+const isVerbose = argv.verbose || argv.v;
+const root = path.dirname(require.main.filename);
+
+// prepare linter service
+const config = {};
+Object.assign(config, getConfigPath(argv));
+const linter = new Linter(config);
+
+// prepare lookup
+const lintDirectories = argv._;
+
+const dirsToTemplates = lintDirectories.reduce((templates, directory) => {
+  const dirTemplates = walkSync(directory)
+    .filter(file => path.extname(file) === '.hbs')
+    .map(file => path.join(directory, file));
+
+  templates.push(...dirTemplates);
+
+  return templates;
+}, []);
+
+const templates = [].concat.apply([], dirsToTemplates);
```

Comment:
is this line necessary ? I thought that the reduce will produce the final flat array ?